src/MeshlessFV/MfvMusclSimulation.cpp

```diff
@@ -148,12 +143,13 @@ void MfvMusclSimulation<ndim>::MainLoop(void)
 
 
 
-  // Calculate terms due to self-gravity
-  if (mfv->self_gravity == 1) {
+  // Calculate terms due to self-gravity / stars
+  if (mfv->self_gravity == 1 || nbody->Nnbody > 0) {
```

Comment:
So I realised I said something stupid, so forget entirely about this comment. The force of the stars onto SPH particles is always included, even when self-gravity is not included. This is because also in UpdateAllSphHydroForces we call ComputeStarGravForces